dbsync-central/application.prod.properties

```diff
@@ -0,0 +1,182 @@
+
+
+# *********************** Logging Configuration ************************************************************************
+#
+# Uncomment this to use console logging otherwise defaults to the logback.xml file located on the classpath which is
+# configured to log to ${eip.home}/logs/openmrs-eip.log
+logging.config=file:logback-spring.xml
+#logging.config=classpath:logback-console.xml
+
+# Logging level for the application loggers (all built-in routes and classes)
+openmrs.eip.log.level=INFO
+
+# Logging level for ONLY application class loggers, feel free to add more logger configurations
+logging.level.org.openmrs.eip=${openmrs.eip.log.level}
+logging.level.receiver-main=${openmrs.eip.log.level}
+logging.level.inbound-db-sync=${openmrs.eip.log.level}
+logging.level.inbound-logger=${openmrs.eip.log.level}
+logging.level.inbound-error-handler=${openmrs.eip.log.level}
+logging.level.receiver-retry=${openmrs.eip.log.level}
+logging.level.inbound-complex-obs-sync=${openmrs.eip.log.level}
+logging.level.message-processor=${openmrs.eip.log.level}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Tomcat Configuration *************************************************************************
+#
+# Server HTTP port, the H2 database console is exposed at this port
+server.port=${server_port}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Receiver camel routes location ***************************************************************
+#
+# Location of the Camel routes xml files
+camel.springboot.xmlRoutes=file:routes${file.separator}*.xml
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** OpenMRS Webapp Configuration *****************************************************************
+#
+# The purpose of this config is so that the receiver sync app can make a rest call to trigger search index rebuild
+# it processes a payload for an indexed entity e.g. person_name, person_attribute, patient_identifier etc.
+# It's highly recommended that you create a specific user account for use here
+#
+# The base URL to the receiving OpenMRS webapp
+openmrs.baseUrl=http://openmrs-central:8080/openmrs
+
+# The username for the account to use to log into the receiver OpenMRS instance
+openmrs.username=${openmrs_username}
+
+# The password for the account to use to log into the receiver OpenMRS instance
+openmrs.password=${openmrs_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** H2 configuration *****************************************************************************
+#
+# Whether to enable the console
+#spring.h2.console.enabled=true
```

Comment:
Shouldn't this be enabled?

Reply:
I think no. We are not using H2 anymore.

Reply:
Ok